app/models/manageiq/providers/ibm_power_vc/inventory/parser/cloud_manager.rb

```diff
@@ -3,6 +3,9 @@ def parse_vm(vm, hosts)
     super
     server = persister.vms.find_or_build(vm.id.to_s)
 
+    operating_system = persister.operating_systems.find(server)
+    operating_system.name = vm.attributes['operating_system'].to_s
```

Comment:
If we don't have an attributes['operating_system'] this will set operating_system.name = "" maybe check for `operating_system.name = vm.attributes['operating_system'].to_s if vm.attributes.key?('operating_system')